yetanotherco/aligned_layer
# Fix: Bump `eigensdk-go` version

# Bump eigensdk-go version 

## Description

Closes #1468 . Bumps the `eigensdk-go` version from `v1.1.12` -> `v1.1.13` . 

Note, `v1.1.12`  contained a library for parsing .yaml and .json files. This library was used in Aligned for starting the variouscomponents of the network. This library was removed in `v1.1.13`. To work around this breaking change the library was re-implemented within the Aligned Layer repo within `core/utils/config_utils.go`, the library was ported from this [branch](https://github.com/Layr-Labs/eigensdk-go/blob/v0.1.12_blsagg_window/utils/utils.go`) of the eigen-sdk repository.

## To Test:
- Run a local devnet and verify proofs land:
1. `make anvil_start_with_block_time`
2. `make aggregator_start`
3. `make operator_full_registration CONFIG_FILE=config-files/config-operator-1.yaml` 
4. `make operator_start CONFIG_FILE=config-files/config-operator-1.yaml`
5. `make batcher_start_local`
6. `make telemetry_full_start`

Once you have the system running, send proofs

7. `make batcher_send_burst_groth16`
8. Go to jaeger `http://localhost:16686/search`
9. You should see the traces correctly created and task verified
<img width="939" alt="image" src="https://github.com/user-attachments/assets/e600fa00-5c11-4fdc-b028-5641890d86a8">


## Type of change

Please delete options that are not relevant.


## Checklist
